tests/meson.build

```diff
@@ -2,6 +2,7 @@ if check_dep.found()
   if (signing_plugin == 'unthreaded')
     subdir('check')
   else
+    subdir('check')
```

Comment:
You need to change the default setting to `threaded` in `mseon_options.txt` as well to make sure we run the tests (temporarily) with the code in the plugin.